src/Microsoft.Sbom.Api/Providers/FilesProviders/CGScannedExternalDocumentReferenceFileProvider.cs

```diff
@@ -53,8 +53,11 @@ public override bool IsSupported(ProviderType providerType)
     {
         if (providerType == ProviderType.Files)
         {
-            Log.Debug($"Using the {nameof(CGScannedExternalDocumentReferenceFileProvider)} provider for the files workflow.");
-            return true;
+            if (Configuration.FilesList?.Value == null && string.IsNullOrWhiteSpace(Configuration.BuildListFile?.Value))
```

Comment:
Would it make sense to change lien 54 to something like this, and ship the check for the FileList?
if (providerType == ProviderType.File || Configuration.ManifestToolAction.Value == ManifestToolActions.Consolidate)